components/VenueMapCard.tsx

```diff
@@ -95,7 +95,7 @@ export default function VenueMapCard({
       <div
         className={clsx(
           "flex justify-center text-red-400",
-          venuesPath.includes(venue.name) && "text-blue-400"
+          venuesPath.includes(venue) && "text-blue-400"
```

Comment:
you could add another guard here something like this:

```
const isNullish = (value: unknown): value is null | undefined => {
  return value === null || value === undefined;
}
```

Reply:
Would be good to talk through this guarding idea it at some point.

Reply:
ok sounds good.